scraper/src/strategies/default_strategy.py

```diff
@@ -256,7 +256,7 @@ def _handle_default_values(hierarchy, content, selectors, levels):
         return hierarchy, content
 
     def _get_nodes_per_global_level(self, selectors, levels):
-        for level in selectors.keys():
+        for level in list(selectors.keys()):
```

Comment:
```suggestion
        for level in selectors:
```

Reply:
Not compliant with python 2.7, Will keep it in our backlog when we will stop supporting 2.7. Thanks for sharing.